video-processing-service/package.json

```diff
@@ -13,28 +13,26 @@
         "test:watch": "jest --watch",
         "test:coverage": "jest --coverage"
     },
-    "keywords": [],
-    "author": "",
-    "license": "ISC",
-    "description": "",
     "dependencies": {
         "clear": "^0.1.0",
         "express": "^4.18.2",
-        "ffmpeg": "^0.0.4"
+        "fluent-ffmpeg": "^2.1.3",
+        "has-symbols": "^1.0.3"
     },
     "devDependencies": {
         "@types/express": "^4.17.21",
+        "@types/fluent-ffmpeg": "^2.1.27",
         "@types/jest": "^29.5.11",
         "@types/node": "^20.10.6",
         "@typescript-eslint/eslint-plugin": "^6.16.0",
         "@typescript-eslint/parser": "^6.16.0",
+        "esbuild": "^0.24.0",
         "eslint": "^8.56.0",
         "eslint-config-prettier": "^9.1.0",
-        "globals": "^13.24.0",
         "jest": "^29.7.0",
         "prettier": "^3.1.1",
         "ts-jest": "^29.1.1",
-        "tsx": "^4.7.0",
+        "tsx": "^4.19.2",
```

Comment:
`issue` `blocking` `question`:  Do you notice this update? Did you check the change logs?